extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/AbstractJpaOperations.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.hibernate.orm.panache.common.runtime;
 
+import static io.quarkus.hibernate.orm.runtime.PersistenceUnitUtil.*;
```

Comment:
wildcard imports are prohibited

Reply:
*grumble* *grumble*  between my IDE formatting and the build formatting that should've never made it in.  :)